File: WorldFeed/Client/src/app/core/Public/Models/Interfaces/DTO/IGetMessageDTO.ts
import {IMessageEventDTO} from "./Events/IMessageEventDTO";
import {IApp} from "../IApp";
import Dictionary from "../../../../../c#-objects/TypeScript.NET-Core/packages/Core/source/Collections/Dictionaries/Dictionary";

export interface IGetMessageDTO {
  messageEvent: IMessageEventDTO;
  apps: Dictionary<number, IApp>;
}

File: Sharebook/Server/src/Services/Message/Sharebook.Message.API/DTO/IGetMessageDTO.cs
namespace Sharebook.Message.DTO
{
    using System.Collections.Generic;

    using Sharebook.Common.Public.Models.Interfaces;
    using Sharebook.Common.Public.Models.Interfaces.DTO.Events;

    public interface IGetMessageDTO
    {
        IMessageEventDTO MessageEvent { get; set; }

        Dictionary<long, IApp> Apps { get; set; }
    }
}
